src/ui/util/use-hover.js

```diff
@@ -0,0 +1,23 @@
+import { useEffect, useState } from 'react';
```

Comment:
No need to do this now, but I suspect these should end up with their own folder or another organizational method rather than being dumped in `util`.